src/ui/main_window.ui

```diff
@@ -97,32 +97,24 @@
     <addaction name="dactionScriptEditor"/>
     <addaction name="dactionShowErrors"/>
     <addaction name="dactionShowMap"/>
+    <addaction name="dactionToggleLogging"/>
+    <addaction name="dactionToggleReplay"/>
+    <addaction name="dactionToggleEmergencyStop"/>
     <addaction name="dactionInputLine"/>
     <addaction name="dactionNotepad"/>
     <addaction name="dactionIRC"/>
     <addaction name="dactionPackageManager"/>
     <addaction name="dactionReplay"/>
     <addaction name="dactionModuleManager"/>
     <addaction name="dactionPackageExporter"/>
-    <addaction name="dactionToggleTimeStamp"/>
-    <addaction name="dactionToggleReplay"/>
-    <addaction name="dactionToggleLogging"/>
-    <addaction name="dactionToggleEmergencyStop"/>
    </widget>
```

Comment:
It seems like you kept these 3 options in the same menu, but changed their position inside. Deliberately or no?